app/models/concerns/ats/workable/job_details.rb

```diff
@@ -12,6 +12,12 @@ def fetch_id(job_data)
         job_data['shortcode']
       end
 
+      def fetch_posting_url(job_data, company_id)
+        # build long-form posting url that includes company_id
```

Comment:
If you put the comment above the method then it will generate docs correctly. I like having these in and would be keen to include these on more methods. FAO also @patanj101

Reply:
Fixed!